make/test.mk

```diff
@@ -1,20 +1,28 @@
+############################################################
+#
+# (local) Tests
+#
+############################################################
+
 .PHONY: test
 ## runs the tests without coverage and excluding E2E tests
 test:
 	@echo "running the tests without coverage and excluding E2E tests..."
 	$(Q)go test ${V_FLAG} -race $(shell go list ./... | grep -v /test/e2e) -failfast
+
 	
-.PHONY: test-ci
-# runs the tests and uploads the coverage report on codecov.io
-# DO NOT USE LOCALLY: must only be called by OpenShift CI when processing new PR and when a PR is merged! 
-test-ci: test-with-coverage upload-codecov-report
```

Comment:
Why are you removing `test-ci`?

Reply:
I thought it's still being used? But apparently it's not?
I'm confused... How do we upload codecov reports then if test-ci is not used?

Btw, regarding codecov reports. Ideally we should collect coverage for both unit and e2e tests. Not sure how we could do it practically... (cc: @xcoulon )

Reply:
It was used before. Now it's removed.

Reply:
But where/when we report to codecov now?

Reply:
Also I see we still have it in member repo. Can we align our repos?

Reply:
@alexeykazakov: we are using two make targets instead of it, so I think we can make it. Look at https://github.com/openshift/release/pull/4266

Reply:
> Also I see we still have it in member repo. Can we align our repos?

I will remove it

Reply:
> we are using two make targets instead of it
Ah.. right. Missed that.
So, looks good!

Reply:
@alexeykazakov yes, we discussed with @dipak-pawar and realized that the `test-ci` goal had little or no benefit, and we could just set the command to execute to `make test-with-coverage upload-codecov-report` in the job, which gives the same result.